root/usr/share/fdi/facts/ethtool.rb

```diff
@@ -9,7 +9,8 @@
 require 'facter'
 
 Facter.add(:ethtool, :timeout => 10) do
-  confine :kernel => "Linux"
+  confine kernel: "Linux"
+  confine :networking { |value| value && value['interfaces'] }
```

Comment:
The syntax is incorrect:

```console
$ facter --custom-dir facts ethtool
2021-05-03 13:57:20.511128 ERROR puppetlabs.facter - error while resolving custom facts in /home/ekohl/facts/ethtool.rb: /home/ekohl/facts/ethtool.rb:13: syntax error, unexpected '{', expecting `end'
  confine :networking { |value| value && value['inte...
                      ^
/home/ekohl/facts/ethtool.rb:13: syntax error, unexpected '}', expecting end-of-input
... value && value['interfaces'] }
...                              ^

```

This works for me:

```suggestion
  confine :networking do |value|
    value && value['interfaces']
  end
```

Reply:
Yeah that I spotted, but after I fixed it locally I cannot see any fact being added. Let me push.